tests/utils.py

```diff
@@ -105,14 +107,6 @@ class mock_users:
     def __init__(self):
         pass
 
-    # Have to add the categoies before loading events
```

Comment:
Any reason we're pulling these out? I think theres a foreign key constraint here 🤔

Reply:
This function is for mocking users, not events